server/seeders/20240321184138-populateKeyForAtAndBrowser.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+    async up(queryInterface) {
+        await queryInterface.sequelize.query(`
+          UPDATE
+            "At"
+          SET
+            "key" = CASE
+              WHEN id = 1 THEN 'jaws'
+              WHEN id = 2 THEN 'nvda'
+              WHEN id = 3 THEN 'voiceover_macos'
+              ELSE null
+            END;
+        `);
+
+        await queryInterface.sequelize.query(`
+          UPDATE
+            "Browser"
+          SET
+            "key" = CASE
+              WHEN id = 1 THEN 'firefox'
+              WHEN id = 2 THEN 'chrome'
+              WHEN id = 3 THEN 'safari_macos'
+              ELSE null
+            END;
+        `);
+    },
+
+    async down(queryInterface) {
+        await queryInterface.removeColumn('At', 'key');
+        await queryInterface.removeColumn('Browser', 'key');
```

Comment:
The reverse here is probably to just set `At.key` and `Browser.key` to null for the rows. You could do that with sequelize's `bulkUpdate`

Reply:
Ok I just used the `bulkUpdate` and it worked great!